src/silo/query_engine/bad_request.h

```diff
@@ -7,7 +7,8 @@
 
 #define CHECK_SILO_QUERY(condition, ...)      \
    do {                                       \
-      if (!(condition)) {                     \
+      bool condition_bool = condition;        \
```

Comment:
For my understanding: Is this to make sure that `condition` is actually a `bool`? And if it's not, the compiler will complain now (instead of maybe implicitly applying an overloaded `!` operator to it)?